src/core/dbus-unit.c

```diff
@@ -839,6 +871,39 @@ static int property_get_refs(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_transient_owners(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        Unit *u = ASSERT_PTR(userdata);
+        int r;
+
+        assert(bus);
+        assert(reply);
+
+        r = sd_bus_message_open_container(reply, 'a', "(utt)");
+        if (r < 0)
+                return r;
+
+        UnitTransientOwner *owner;
+        HASHMAP_FOREACH(owner, u->transient_owners) {
+                (void) pidref_acquire_pidfd_id(&owner->pidref);
+
+                r = sd_bus_message_append(reply, "(utt)",
+                                          owner->pidref.pid, owner->pidref.fd_id,
```

Comment:
shouldn't this also send out the pidfd, so that if the caller wants to track or do something with the list of owners, can do so directly without having to re-create it (and recheck it)?